e2e/integrations/integration.steps.ts

```diff
@@ -183,6 +183,15 @@ class IntegrationSteps {
   public getStepsArray (): P<any> {
     const stepFactory = new StepFactory();
     const steps = this.world.app.getElementsByClassName('parent-step');
+    const navBar = this.world.app.getElementByClassName('nav-pf-vertical');
+
+    navBar.isDisplayed().then((visible) => {
+      if (visible) {
+        log.warn('Navigation bar is not hidden.');
+        const navBarToggle = this.world.app.getElementByClassName('navbar-toggle');
+        navBarToggle.click();
+      }
+    });
 
```

Comment:
I guess now we're permanently hiding the nav bar due to removal of this line https://github.com/syndesisio/syndesis-e2e-tests/pull/44/commits/4de6a0d584c7c3c460a385a6614dcc5156f5c479#diff-b5c6e4c77b8cafd1a1ce1d7a0bc7b3e6R204. Either add back that click with condition also or we need to sanitize navigation bar navigation to check if visible before clicking.